### PR TITLE
set source of pip

### PR DESCRIPTION
**Pull request by XuHugo**
set source of pip in docker-agent
*You can modify the pip source by yourself, and then update the pip source with PIP parameters

Signed-off-by: XuHugo <xq-310@163.com>

### CODE REVIEW

**Comment by XuHugo** [comment]
https://github.com/hyperledger/cello/wiki/Debug-cello-by-container

**Review by yeasy — APPROVED** [review]

File: build_image/docker/agent/ansible/Dockerfile.in [hunk 1]
```diff
@@ -10,8 +10,9 @@ ARG uid=1000
 ARG gid=1000
 
 RUN apt-get update                                          && \
+    apt-get upgrade     &&\
     apt-get install -y bash curl python-dev sudo               \
-    python-pip build-essential openssh-client libffi-dev       \
+    python3-pip build-essential openssh-client libffi-dev       \
     libssl-dev                                              && \
     pip install --upgrade pip ansible pyyaml                && \
```
Comment (yeasy) [inline build_image/docker/agent/ansible/Dockerfile.in:17]: Should we use pip3 here?

Comment (XuHugo) [reply build_image/docker/agent/ansible/Dockerfile.in:16]: If you don't use it, it will prompt that python-pip is not found.

**Pull request merged**